### DISCUSSION TITLE
Rectified stereo images

### DISCUSSION THREAD

**Discussion by banafshebamdad** [realsenseai/realsense-ros#2860] [Q&A] [answered]
Hi everyone,
I've recently recorded ROS bags using a RealSense D435i camera containing /camera/infra1/image_rect_raw and /camera/infra2/image_rect_raw. I'm wondering if these stereo frames are rectified.
Thanks,
Banafshe

**Comment by MartyG-RealSense** [comment, marked as answer]
Hi @banafshebamdad  The Y8 infrared format is rectified.  The Y16 infrared format is unrectified.  

If you are recording using the RealSense ROS wrapper then the infrared frames will be Y8 rectified, as the option to use Y16 infrared is only supported in the librealsense SDK, in tools such as the RealSense Viewer.

**Reply by banafshebamdad** [reply]
Thanks a lot!
